src/docfx/build/Build.cs

```diff
@@ -166,13 +166,18 @@ void ValidateBookmarks()
                     return DependencyMap.Empty;
                 }
 
+                var (groupId, outputPath) = GetOutputPath(file, monikers);
```

Comment:
>groupId [](start = 21, length = 7)

what's group?

Reply:
group id is the first 8 characters of the hash of the monikers joined by whitespace.